bin/rialto-parachain/node/src/service.rs

```diff
@@ -502,8 +502,8 @@ pub async fn start_node(
 					keystore,
 					force_authoring,
 					slot_duration,
-					// We got around 500ms for proposing
-					block_proposal_slot_portion: SlotProportion::new(1f32 / 24f32),
+					// We got around 1000ms for proposing
```

Comment:
This has caused problems for me - parachain was unable to produce any blocks within `500ms`, so I've increased it to `1s`.